rake/build/environment.rb

```diff
@@ -79,8 +79,8 @@
 pipeopts 'st2mistral' do
   checkout true
   envpass :giturl,  'https://github.com/StackStorm/mistral', from: 'ST2MISTRAL_GITURL'
-  envpass :gitrev,  'st2-1.3.2',                             from: 'ST2MISTRAL_GITREV'
+  envpass :gitrev,  'master',                                from: 'ST2MISTRAL_GITREV'
```

Comment:
Got it as part of this https://github.com/StackStorm/st2-packages/pull/245. 

Please revert changes to this file (Done in 32188e5). I do think fixing environment.rb is needed but let's get that as part of some other PR.